marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/FlowController.java

```diff
@@ -379,100 +370,13 @@ public StepModel createStep(String flowName, Integer stepOrder, String stepId, S
         }
 
         if (existingStep != null && existingStep.isEqual(step)) {
-            return transformStepToWebModel(existingStep);
+            return existingStep;
         }
 
         flowManager.saveFlow(flow);
-        return transformStepToWebModel(step);
-    }
-
-    private List<StepModel> getStepsToWebModel(String flowName) {
```

Comment:
If the step is going to look something like the one below, I don't think it's needed. We allowed creating custom steps in QS, hence StepModel contained "modulePath"  (which was not in Step) which was used to create a step definition and a step in the backend. Since we won't do that, I believe it wouldn't be necessary. We can include it if there is a need
```
    "1" : {
      "name" : "mapItem",
      "options" : {
        "mapping" : {
          "name" : "mapItem"
        }
      },
      "stepDefinitionName" : "entity-services-mapping",
      "stepDefinitionType" : "MAPPING"
    }
```

Reply:
Sounds good!